PLAT-129963: Update "Hello, Enact" sandstone dependency (#166)
enactjs/docs: src/pages/docs/tutorials/tutorial-hello-enact/basics/index.md

```
You'll notice that by introducing `<div>` we no longer have valid JavaScript! In fact, it looks a lot like valid HTML. That's because React introduces [JSX](https://reactjs.org/docs/jsx-in-depth.html), which is a JavaScript syntax extension. In order to make JSX runnable by the browser, it has to be converted to JavaScript. With Enact, this is handled during the build process using [webpack](https://webpack.js.org) and [babel](https://babeljs.io/). More on this [later](#running-the-app).

> You may wonder why import `React` even if it's not explicitly used. That is because it's used by transpiled JavaScript which was JSX. Thus, importing `React` was necessary to use JSX. After React 17, these are [improved](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-different-in-the-new-transform), so no longer needed to import it. You can remove the code (`import React from 'react'`) since our app uses React 17.

Reviewer: Back in the time, we need to import `react` module in order to use JSX. But as of now, we no longer needed to import since React 17 introduced [the new JSX transform](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html).

Author: Thank you :)

> The `index.js` provided by the dev tools allows the `App` component to be rendered into the DOM or imported into another
> component.

### `import` and ReactDOM

We'll use `import` to bring our dependencies. The [`import` statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) is a new feature introduced with ES6 modules ([spec](https://262.ecma-international.org/6.0/#sec-imports)).

> Rendering is handled by `react-dom` module. [ReactDOM](https://reactjs.org/docs/react-dom.html) provides the means to transform a React component tree into a DOM tree. You'll primarily be interested in the [`render()` method](https://reactjs.org/docs/react-dom.html#render).

Reviewer: ```suggestion
Let's import `react-dom` module for rendering our App. [ReactDOM](https://reactjs.org/docs/react-dom.html) provides the means to transform a React component tree into a DOM tree. You'll primarily be interested in the [`render()` method](https://reactjs.org/docs/react-dom.html#render).
```

Author: 🆗
